code/game/objects/structures/crates_lockers/closets/secure/security.dm

```diff
@@ -12,6 +12,7 @@
 	new /obj/item/storage/backpack/duffelbag/captain(src)
 	new /obj/item/clothing/neck/cloak/cap(src)
 	new /obj/item/clothing/neck/petcollar(src)
+	new	/obj/item/clothing/suit/armor/vest/capcarapce/jacket(src)
```

Comment:
this gives captain the same issue HOS has, where he has multiple pieces of equally high-grade armor in one locker for cosmetic purposes.

Reply:
what do you propose then?